.kube/app/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM php:8.4-fpm
 
 ENV PHP_VERSION 8.4
+ENV IMAGICK_VERSION 3.7.0 # This should be bumped to the latest version and
```

Comment:
seems like part of the comment is missing

Reply:
updated